app/Exceptions/Handler.php

```diff
@@ -108,9 +113,24 @@ public function render($request, Exception $e)
                 }
             }
 
-            return response()->json([
+            $response = response()->json([
                 'errors' => $errors
             ], $statusCode, $headers);
+
+            // In the circumstance where an exception is raised
+            // before the lumen request cycle reaches the middleware stage,
+            // it is necessary to force the inclusion of the CORS headers.
```

Comment:
:( I guess we should plan to rework the Multisite stuff into middleware in a future iteration. It's kind of gross that this can ever happen, but clearly out of scope for this PR.